### PR TITLE
fix: install dependencies issues

### PR DESCRIPTION
**Pull request by cr7258**
### Description

This commit addresses the problem with these issuse:

1. Missing wget on centos and fedora system.

image

2. luarocks was installed in /usr/local/bin，make sure /usr/local/bin is included in the system default PATH variable. 

image

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: utils/install-dependencies.sh [hunk 1]
```diff
@@ -46,7 +46,7 @@ function install_dependencies_with_aur() {
 function install_dependencies_with_yum() {
     sudo yum install -y yum-utils
 
-    local common_dep="curl git gcc openresty-openssl111-devel unzip pcre pcre-devel openldap-devel"
+    local common_dep="curl wget git gcc openresty-openssl111-devel unzip pcre pcre-devel openldap-devel"
```
Comment (tzssangglass) [inline utils/install-dependencies.sh:49]: It seems we don't need to modify here?

The `common_dep` here is for the CI runtime environment and only needs to be installed if it is not present in the CI.

Comment (cr7258) [reply utils/install-dependencies.sh:49]: utils/linux-install-luarocks.sh will execute wget command to get luarock tar file.
```bash
# utils/install-dependencies.sh
function install_luarocks() {
    curl https://raw.githubusercontent.com/apache/apisix/master/utils/linux-install-luarocks.sh -sL | bash -
}
```

```bash
# utils/linux-install-luarocks.sh
wget https://github.com/luarocks/luarocks/archive/v"$LUAROCKS_VER".tar.gz
```

Comment (tzssangglass) [reply utils/install-dependencies.sh:49]: ok

File: utils/linux-install-luarocks.sh [hunk 2]
```diff
@@ -53,5 +53,13 @@ if [ -d ${OPENRESTY_PREFIX}/openssl111 ]; then
     OPENSSL_PREFIX=${OPENRESTY_PREFIX}/openssl111
 fi
 
+FOUND_PATH=$(echo "${PATH}" | grep -oP '(?<=:|)/usr/local/bin(?=:|)') || true
+if [[ "${FOUND_PATH}" == "" ]]; then
+   echo "the path /usr/local/bin is not included in the system default PATH variable, add PATH variable in ~/.bashrc."
+   echo "export PATH=$PATH:/usr/local/bin" >> ~/.bashrc
```
[the remark this answers is not in the document]
Comment (cr7258) [reply utils/linux-install-luarocks.sh]: ok

**Review by spacewander — APPROVED** [review]

**Review by tzssangglass — APPROVED** [review]

**Pull request merged**